hello_sqlite/config/host.exs

```diff
@@ -0,0 +1,14 @@
+import Config
+
+# Add configuration that is only needed when running on the host here.
+
+config :hello_sqlite,
+  ecto_repos: [HelloSqlite.Repo]
+
+config :hello_sqlite, HelloSqlite.Repo,
+  database: "/tmp/database.db",
```

Comment:
```suggestion
  database: "/data/database.db",
```

Shouldn't this be in `/data` so that it's around after a reboot?

Reply:
oops yep. i missed this

Reply:
@fhunleth actually on second look, this is the `host.exs` config so `/data` won't work. The `target.exs` file points to `/data`